src/restapi/mm_routes.ts

```diff
@@ -0,0 +1,49 @@
+import fs from 'fs';
+
+import express, {Request, Response} from 'express';
+import {AppCall} from 'mattermost-redux/types/apps';
+
+import app from '../app/app';
+
+import {jsonStoreFileName} from '../app/constants';
+
+const router = express.Router();
+
+router.post('/createform', async (req: Request, res: Response) => {
+    const appCall: AppCall = req.body;
+    if (appCall.type === 'form') {
+        const createForm = app.calls.createForm(appCall.context.post.message);
+        res.json(createForm);
+    } else {
+        const message = app.createTicketFromPost(appCall);
+        res.json({});
+    }
+});
+
+router.get('/manifest.json', (req: Request, res: Response) => {
+    res.json(app.manifest);
+});
+
+router.get('/bindings', (req: Request, res: Response) => {
+    res.json(app.bindings.getBindings());
```

Comment:
I think this would read better

```suggestion
    res.json(app.getBindings());
```

Reply:
I wonder if this function needs to belong to the App class? It returns a list of hardcoded location bindings to a call.  It could almost reside in `src/restapi/` and be imported via `import {getBindings} from './bindings'`.  What do you think?

Reply:
bindings has been moved into its own folder `bindings/` similar to the `forms/` folder.